### DISCUSSION TITLE
Can you power the led with the rpi 5v pin?

### DISCUSSION THREAD

**Discussion by Cereal916** [thomst08/OctoLight#11] [Q&A]
From what I can tell in the tutorials I've found, people are powering the led strip with a separate power supply, and using the rpi 5v pin to power a relay. Is there a reason you can't use the second 5v pin on the rpi to power the led strip?

**Comment by thomst08** [comment]
Hey @Cereal916,

This is a little out of my area of knowledgeable and isn't something I've done myself or have any connections to any tutorials out there.
But... When powering LED's (specifically the 5V LEDs), they can draw a fair bit of power.  There is no reason why you couldn't pull the power from the Pi to do this, however, I wouldn't as I could see it ether pulling too much current from the Pi and therefor causing it to shutdown or worst, this would also be an issue depending on what you have powering the Pi as not all USB chargers are the same, but it depends on what your using.  You might also find it limits the amount of power from the 5V pin, meaning it might work fine for a meter strip, but as soon as you add more you wont have enough current to power the whole strip.

Basically in my mind, you could damage your Pi due to pulling too much current from the system, its saver to have its own power source depending on how complex your setup is, but if anyone has a better understanding, please correct me.
